live-examples/css-examples/fonts/font-stretch.css

```diff
@@ -0,0 +1,13 @@
+@font-face {
+    font-family: 'LeagueMonoVariable';
+    src: local('LeagueMonoVariable'),
+         url('../../../media/fonts/LeagueMonoVariable.ttf') format('truetype');
```

Comment:
I think the url can just be '/media/fonts/LeagueMonoVariable.ttf', can't it? That's how we are referencing fonts in the other `@font-face` rules, like https://github.com/mdn/interactive-examples/blob/master/live-examples/css-examples/fonts/font-variation-settings.css.